src/oc/ocWrapper.ts

```diff
@@ -85,10 +86,15 @@ export class Oc {
     }
 
     public async getAllKubernetesObjects(namespace?: string, executionContext?: ExecutionContext): Promise<KubernetesObject[]> {
-        const result = await CliChannel.getInstance().executeTool(
-            Oc.getKubernetesObjectCommand('all', namespace),
-            undefined, true, executionContext);
-        return JSON.parse(result.stdout).items;
+        try {
+            const result = await CliChannel.getInstance().executeTool(
+                Oc.getKubernetesObjectCommand('all', namespace),
+                undefined, true, executionContext);
+            return JSON.parse(result.stdout).items;
+        } catch (err) {
+            window.showErrorMessage(`Cannot list HELM Releases: ${err}`);
```

Comment:
These are not helm releases, they are Kubernetes objects

Reply:
Oh. Sure.